### PR TITLE
update Vagrant instructions. E.g. cucumber => behave

### PR DESCRIPTION
**Pull request by mtmail**
* documentation still pointed to cucumber tests
* we no longer recommend centos for Vagrant development
* we use Ubuntu 16 in Vagrant, not 14

### CODE REVIEW

File: VAGRANT.md [hunk 1]
```diff
@@ -65,13 +61,31 @@ see Nominatim in action on [locahost:8089](http://localhost:8089/nominatim/).
 You edit code on your host machine in any editor you like. There is no need to
 restart any software: just refresh your browser window.
 
+Note that the webserver uses files from the /build directory. If you change
+files in Nominatim/lib for example you first need to copy them into the
+/build directory.
```
Comment (lonvia) [inline VAGRANT.md]: Nominatim/lib is fine. /website and /utils are copied and they should be refreshed in the build directory by rerunning cmake.

File: VAGRANT.md [hunk 2]
```diff
@@ -97,13 +111,7 @@ To run specific tests you can add tags just before the `Scenario line`, e.g.
...
 and then
 
-    NOMINATIM_SERVER=http://localhost:8089/nominatim lettuce -t bug-34
-
-
-## Running unit tests
-
-    cd ~/Nominatim/tests/php
-    phpunit ./
+    behave -DBUILDDIR=/home/vagrant/build/ --tags @bug-34
```
Comment (lonvia) [inline VAGRANT.md]: With behave you can now just state the line in the file to run a single test, e.g like that:

    behave -DBUILDDIR=/home/vagrant/build/ features/api/reverse.feature:34

Very useful feature.

Comment (mtmail) [reply VAGRANT.md]: Nice, just like Ruby's rspec

**Comment by mtmail** [comment]
Added your feedback, thanks a lot!

**Pull request merged**